apps/api/src/app/groups/groups.service.ts

```diff
@@ -142,6 +142,16 @@ export class GroupsService {
 
         await this.groupRepository.save(group)
 
+        if (treeDepth) {
```

Comment:
I think we can merge this if block and the other one at line 126. What do you think?

Reply:
Yes, true. Thanks! 👍